server/runtime/src/main/java/org/infinispan/server/configuration/security/RealmConfiguration.java

```diff
@@ -15,9 +15,11 @@
  */
 public class RealmConfiguration implements ConfigurationInfo {
    static final AttributeDefinition<String> NAME = AttributeDefinition.builder("name", null, String.class).build();
+   static final AttributeDefinition<Integer> CACHE_MAX_SIZE = AttributeDefinition.builder("cacheMaxSize", 16).build();
```

Comment:
Isn't 16 kind of small?

Reply:
It seems so. If I understand it correctly, this cache is mapped 1:1 to user credentials in the realm. If the entries are small, we could maybe up this number?

Reply:
To be honest, I was thinking that many systems will only have a handful of application users.